server/services/visitNotificationsService.ts

```diff
@@ -20,13 +19,6 @@ export default class VisitNotificationsService {
     return orchestrationApiClient.getNotificationCount(prisonId)
   }
 
-  async getVisitRequestCount(username: string, prisonId: string): Promise<VisitRequestsCountDto> {
```

Comment:
Moved this to new `VisitRequestService` as there will be more request-related stuff coming.